### DISCUSSION TITLE
Database messed up?

### DISCUSSION THREAD

**Discussion by Wonderwuzi2000** [RhetTbull/osxphotos#797] [Q&A]
I'm running MacOS 13 Beta (maybe not a good idea) with iCloud Sync (resized photos).
Now when i use osxphotos and run **osxphotos persons** the result for a certain person (let’s say the name is „xyzzy“) is 9.
When i use **osxphotos query -person „xyzzy“ -add-to-album „Test“** I only get 8 pictures.
I’m confused - could you give me some advice?

**Comment by RhetTbull** [comment]
I've tested osxphotos on a test database from an early macOS 13 beta and verified it could read the database but without direct access to macOS 13 it's impossible for me to do full testing. It's quite possible there's some Photos database change that osxphotos isn't aware of. 

If you create a Smart Album in Photos for „xyzzy“ what result do you get?

**Comment by Wonderwuzi2000** [comment]
Unfortunately also 8 (which btw seems to be the right amount).
So the total situation is that i've got 76 people recognized (and labeled in "Persons") and 26 of them differ in the amount of photos displayed when I run "osxphotos persons" compared to the photos app on macOS (filtering person by person).
All in all confusing, but no big problem.
Let's see what happens when MacOS 13 is out of beta.
In the meanwhile I will try to lend myself a second Mac PC and see what the result is like, when I try to download everything from iCloud there.

**Reply by RhetTbull** [reply]
OK, it's good that osxphotos is reporting the same number as Photos.  The `osxphotos persons` command lists the total number of photos found with the person which could include photos in the trash or "non-selected" burst images so that could also be a source of difference.
